vsprojects/README.md

```diff
@@ -2,7 +2,7 @@ This directory contains MS Visual Studio project & solution files.
 
 #Supported Visual Studio versions
 
-Currently supported versions are Visual Studio 2013 (our primary focus) and 2010.
```

Comment:
we currently support VS2013 and VS2015 (and VS2015 support will become more important as we figure out C++ compilation with MSVC, so I think it's worth mentioning here)